### ISSUE TITLE
Feature request - more fullscreen option 4:3 aspect options

### ISSUE THREAD

**Issue by arrowgent** [AliveTeam/alive_reversing#1000]
Found in build: [insert build number] Linux
Last working build number: [finding this will greatly reduce time to fix] today's master build 12-23-2020

Description of issue:
[insert description]
F12 fullscreen stretches to widescreen 16:9
would prefer option to toggle 4:3 widescreen with borders

i dont know if SDL supports a 3 toggle switch, this would be great if it does.
F12 = windowed
F12 = fullscreen 16:9
F12 = fullscreen aspect 4:3

if not then some toggle option on commandline? 
```
AliveExeAO -aspect
```

great work thanks for the RELIVE REV

**Comment by paulsapps** [comment]
Yeah this kind of thing should be possible - the pause menu dev options currently has the aspect option for AE - marking this as a feature request because it isn't a regression from the original game or a reimplementation bug.

**Comment by paulsapps** [comment]
I believe @Nemin32 added some keys for this, however there still isn't any in game UI for it.

**Comment by Nemin32** [comment]
Yes,
F10 - Bilinear filtering on / off
F11 - Keep aspect ratio / Stretch
F12 - Fullscreen / Windowed mode
